databases/week2/students_db.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE Class(
+	id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    name varchar(255),
+    begins DATE,
+    ends DATE
+);
+CREATE TABLE Student(
+	id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    name varchar(255),
+    email varchar(255),
+    phone INT,
+    class_id INT NOT NULL,
+    CONSTRAINT FOREIGN KEY (class_id) REFERENCES Class(id)
+);
+CREATE INDEX idx_name ON Student(name);
```

Comment:
index on a VARCHAR column will boost query performance
It's make more sense to find a INT column to create INDEX on it. 
```suggestion
CREATE INDEX idx_name ON Student(id);
```